src/modules/processing/services/loom.js

```diff
@@ -0,0 +1,39 @@
+import { genericUserAgent } from "../../config.js";
+
+export default async function({ id }) {
+    const gql = await fetch(`https://www.loom.com/api/campaigns/sessions/${id}/transcoded-url`, {
+        method: "POST",
+        headers: {
+            "user-agent": genericUserAgent,
+            origin: "https://www.loom.com",
+            referer: `https://www.loom.com/share/${id}`,
+            cookie: `loom_referral_video=${id};`,
+
+            "apollographql-client-name": "web",
+            "apollographql-client-version": "14c0b42",
+            "x-loom-request-source": "loom_web_14c0b42",
+        },
+        body: {
+            force_original: false,
+            password: null,
+            anonID: null,
+            deviceID: null
+        }
```

Comment:
missing JSON.stringify, this will just send "[object Object]"

Reply:
im surprised it works at all then 😭